tests/unit/commands/local/lib/test_sam_api_provider.py

```diff
@@ -1013,125 +985,119 @@ def test_provider_stage_variables(self):
             }
         }
         provider = ApiProvider(template)
-        api1 = Api(path='/path', method='GET', function_name='NoApiEventFunction', cors=None, binary_media_types=[],
-                   stage_name='dev',
-                   stage_variables={
-                       "vis": "data",
-                       "random": "test",
-                       "foo": "bar"
-                   })
+        route1 = Route(path='/path', method='GET', function_name='NoApiEventFunction')
 
-        self.assertIn(api1, provider.apis)
+        self.assertIn(route1, provider.routes)
+        self.assertEquals(provider.api.stage_name, "dev")
+        self.assertEquals(provider.api.stage_variables, {
+            "vis": "data",
+            "random": "test",
+            "foo": "bar"
+        })
 
     def test_multi_stage_get_all(self):
-        template = {
-            "Resources": {
-                "TestApi": {
-                    "Type": "AWS::Serverless::Api",
-                    "Properties": {
-                        "StageName": "dev",
-                        "Variables": {
-                            "vis": "data",
-                            "random": "test",
-                            "foo": "bar"
-                        },
-                        "DefinitionBody": {
-                            "paths": {
-                                "/path2": {
-                                    "get": {
-                                        "x-amazon-apigateway-integration": {
-                                            "httpMethod": "POST",
-                                            "type": "aws_proxy",
-                                            "uri": {
-                                                "Fn::Sub": "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31"
-                                                           "/functions/${NoApiEventFunction.Arn}/invocations",
-                                            },
-                                            "responses": {},
-                                        },
-                                    }
-                                }
+        template = OrderedDict({
```

Comment:
define statically

Reply:
The problem with defining with statically is that when we parse the template, we use the order of the template to determine which stage to process. In python3 by default, dictionaries have order ,but in python2, there is no guarantee.